DiscordWebhooks/DiscordWebhooks.cs

```diff
@@ -59,6 +59,54 @@ public async Task PostMessageAsync(WebhookMessage message)
 
             _ = (HttpWebResponse)(await request.GetResponseAsync());
         }
+
+        public void EditMessage(ulong messageId, WebhookMessage message)
+        {
+            if (message.username != default) { throw new Exception("DiscordWebhooks: username field is not supported for editing messages."); }
```

Comment:
Throwing an exception on these fields being set could complicate generation of discord messages. As this would require a different method/flag for edited messages from new messages.

Removing this would allow for the same methods used to generate the initial message to be used to update existing messages.